docs/vizier/vizier.rst

```diff
@@ -20,11 +20,12 @@ Table Discover
 If you want to search for a set of tables, e.g. based on author name or other keywords,
 the :meth:`~astroquery.vizier.VizierClass.find_catalogs` tool can be used:
 
+.. order is not deterministic
 .. doctest-remote-data::
 
     >>> from astroquery.vizier import Vizier
-    >>> catalog_list = Vizier.find_catalogs('hot jupiter exoplanet transit')
-    >>> for k, v in catalog_list.items():
+    >>> catalog_list = Vizier().find_catalogs('hot jupiter exoplanet transit')
```

Comment:
similar cases exists a lot of other places in this docs. Maybe do a class instance e.g. the usual `vizier = Vizier()` and then use it everywhere instead of Vizier?

Reply:
Sure! It's done :slightly_smiling_face: